src/components/action-bar/action-bar.tsx

```
@@ -132,13 +147,17 @@ export class ActionBar {
                 class={{
                     'is-full-width': this.layout === 'fullWidth',
                     'is-floating': this.layout === 'floating',
+                    'is-shrunk':
+                        !!this.actionBarIsShrunk && !!this.isExpandable,
+                    'can-be-shrunk': !!this.isExpandable,
                 }}
                 role="grid"
             >
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<table>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)